src/Functions/dateDiff.cpp

```diff
@@ -1,3 +1,6 @@
+#include "Core/SettingsEnums.h"
```

Comment:
The reader wonders a) why double quotes are used while l. 4ff uses <> b) what makes both new includes special enough to deserve a newline (l. 3).